### DISCUSSION TITLE
How to remove dividers from vertical menu?

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1257] [Collections (form, grid, menu, message, table, ...)] [answered]
Hello, according to [this](https://github.com/fomantic/Fomantic-UI/issues/1252) I'm using menu for creating a sidebar in one page of my web application:

![menu](https://user-images.githubusercontent.com/5399265/72249886-a7b56300-35fa-11ea-9548-aecfde8c3fd5.png)

I would like to use [vertical menu](https://fomantic-ui.com/collections/menu.html#vertical-menu) with a variation: having items without dividing borders. I mean this:

![menu_after](https://user-images.githubusercontent.com/5399265/72249893-adab4400-35fa-11ea-9e77-222cdd020075.png)

Should Fomantic-UI menu collections offer a variation to users to have the same structure without borders? Perhaps using a new dedicated Fomantic-UI class, for example something like "plain" or "no-dividers"?

In the meanwhile what is the best way to remove dividers? I see that I can reach this unsetting `position: relative` for `.ui.menu .item` selector. Isn't it weird?

In addition this works bad in my case because I'm using nested items in my menu:

```
<div class="ui vertical menu">
    <div class="item">
        <div class="ui transparent icon input">
            <input type="text" placeholder="Model...">
            <i class="search icon"></i>
        </div>
    </div>
    <div class="item">
        RAM
        <a class="sub-item item">
            <div class="ui checkbox">
                <input type="checkbox" tabindex="0" class="hidden">
                <label>8 GB</label>
            </div>
            <div class="ui label">1</div>
        </a>
        <a class="sub-item item">
            <div class="ui checkbox">
                <input type="checkbox" tabindex="0" class="hidden">
                <label>16 GB</label>
            </div>
            <div class="ui label">1</div>
        </a>
    </div>

    <div class="item">
        Screen size
        <a class="sub-item item">
            <div class="ui checkbox">
                <input type="checkbox" tabindex="0" class="hidden">
                <label>13''</label>
            </div>
            <div class="ui label">1</div>
        </a>
        <a class="sub-item item">
            <div class="ui checkbox">
                <input type="checkbox" tabindex="0" class="hidden">
                <label>15''</label>
            </div>
            <div class="ui label">1</div>
        </a>
        <a class="sub-item item">
            <div class="ui checkbox">
                <input type="checkbox" tabindex="0" class="hidden">
                <label>17''</label>
            </div>
            <div class="ui label">1</div>
        </a>
    </div>
</div>
```

![nested](https://user-images.githubusercontent.com/5399265/72250738-99684680-35fc-11ea-950c-f34744d7011b.png)

Do you know a better way to build what I need... and without dividers?

**Comment by lubber-de** [comment]
Try
```css
.ui.vertical.menu .item:before {
    width:0;
}
```

**Comment by smartm0use** [comment]
It works and it is better then my solution.
What about adding a custom class to have it in Fomantic-UI menu collections?

**Comment by lubber-de** [comment, marked as answer]
Well, we already have `secondary vertical menu` which also does not have those lines.

**Comment by smartm0use** [comment]
This is definitely better. Thank you.
